modules/tinymce/changelog.txt

```diff
@@ -1,4 +1,5 @@
 Version 5.7.0 (TBD)
+    Fixed an issue where image dimensions were trying to be calculated for an empty image URL #TINY-6611
```

Comment:
```suggestion
    Fixed an issue where the image dialog tried to calculate image dimensions for an empty image URL #TINY-6611
```
"trying to be" seemed not great to use in the changelog to me (considering we don't allow "being"), so switched things around a bit.

Reply:
cool, thanks